packages/server-wallet/src/utilities/validate-transition.ts

```diff
@@ -1,11 +1,22 @@
 import _ from 'lodash';
-import {SignedState, toNitroState} from '@statechannels/wallet-core';
+import {SignedState, StateWithHash, toNitroState} from '@statechannels/wallet-core';
 import {constants} from 'ethers';
 
 import {logger} from '../logger';
 import {validateTransitionWithEVM} from '../evm-validator';
 import {Bytes} from '../type-aliases';
+import {Channel} from '../models/channel';
 
+export function shouldValidateTransition(incomingState: StateWithHash, channel: Channel): boolean {
+  const {supported, isLedger} = channel;
+  // If we already have the state we should of already validated it
+  const alreadyHaveState = _.some(channel.sortedStates, ['stateHash', incomingState.stateHash]);
+
+  // Ignore older states that may be added via syncing
+  const isOldState = incomingState.turnNum < (supported?.turnNum || 0);
```

Comment:
It seems like we should go beyond not validating, and actually not store states that are older than our latest supported state. Is that accomplished elsewhere?

Reply:
We do not need to store states older than our latest supported state in some situations, e.g., a ledger update